Cleanup Part 2

leshghi opened the pull request:
1. Removed extra sat value dicts
2. Removed AnalysisResult JS class and added the params to ResultBBM
3. Cleaned up switchToAnalysisMode and switchToModellingMode

In draft form because still a WIP but I want @YesugenB and @i-m-foster to take a look at my ElementInspector.js changes to help make sure I didn't break anything/create huge issues for them.

leshghi committed:
Removed extra sat value dicts

Also removed eval: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!

leshghi committed:
Remove AnalysisResult JS class

leshghi committed:
Clean up switchMode functions and remove dead code

leshghi committed:
Fix clearInspector bug when removing actors

i-m-foster commented on leaf-ui/rappid-extensions/ElementInspector.js:
```diff
@@ -339,7 +353,7 @@ var ElementInspector = Backbone.View.extend({
         funcSegments.forEach(
             element => {
```
We could change `element ` to `functionSegment ` here

leshghi replied:
Nice will do in my next commit

i-m-foster reviewed (approved):
It looks good to me! I don't think it should cause any major issues for us

leshghi committed:
Small changes and removing analysis.js global variables

leshghi committed:
Small cleanup of dead code in EVO

Avoiding touching more for now since I don't know much about the tool

leshghi committed:
Reformatted some files

Happy to revert if this causes more merge conflicts than is worth it, but also think its at least worth leaving the index.html file to make it more readable

amgrubb reviewed (approved):
Wow, that's a lot of cleanup. Do you have a formatting tool that corrects the spaces? If so, what do you use/recommend?

leshghi commented:
> Wow, that's a lot of cleanup. Do you have a formatting tool that corrects the spaces? If so, what do you use/recommend?

VSCode does it for you! If you right click and select format document from the menu it will do the whole file. You can also highlight just a portion of code and then choose format selection.
